com.gitbutler.app.metainfo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.gitbutler.app</id>
+  
+  <name>GitButler</name>
+  <summary>Git branch management tool, built from the ground up for modern workflows</summary>
+  
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>FSL-1.0-MIT</project_license>
+
+  <description>
+    <p>
+      Source Code Management system designed to manage your branches, record and backup your work, be your Git client, help with your code and much more.
+    </p>
+  </description>
+  
+  <launchable type="desktop-id">com.gitbutler.app.desktop</launchable>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://blog.gitbutler.com/content/images/2024/01/CleanShot-2024-01-29-at-10.22.46@2x.png</image>
```

Comment:
Do you have a screenshot on Linux?

Reply:
I added the image, but it seems that GitHub adds padding to images with white borders. 😕